Use official repo

mvn23 opened the pull request:
As requested in #12, a PR for my changes to support the official release repo.

* Source repo changed to official
* Version is now taken from release `tag_name`
* $BOARD format updated (for ER* models)

Fixes #11


Edit: version update is pending, I'll leave that op to @mafredri ;)

mvn23 committed:
Support official wireguard-vyatta-ubnt repo

mvn23 committed:
Revert changes to LATEST_FILE filename

Revert changes to LATEST_FILE filename for backwards compatibility.

mvn23 committed:
Change long line split

Change how the long `jq` command is split up across lines in latest_release_for to improve readability

mafredri commented on wireguard.sh:
```diff
@@ -58,8 +54,10 @@ latest_release_for() {
-		| jq -r --arg version "wireguard-${board}" \
-			'sort_by(.created_at) | reverse | .[0].assets | map(select(.name | contains($version))) | .[0] | {name: .name, url: .browser_download_url}'
+		| jq -r --arg version "${board}" \
+			'sort_by(.created_at) | reverse | .[0] | .tag_name as $tag_name \
```
Can you confirm these `\` inside the qutes work as intended on your device? I'm seeing this on my USG:

```
error: syntax error, unexpected INVALID_CHARACTER, expecting '|'
sort_by(.created_at) | reverse | .[0] | .tag_name as $tag_name \
                                                               ^
1 compile error
```

The USG has bash 4.2.37, and bash does support multi-line strings (i.e. `\` is interpreted literally here), but I had still expected this to work 😅.

mvn23 replied:
You're absolutely right! The version I tested does not use the linebreaks, I must have added them afterwards. Now tested without backslashes, then it works. Fix imminent.

mvn23 committed:
Remove backslashes from line breaks

Remove backslashes from line breaks as they are not needed within single quotes

mafredri reviewed (approved):
Perfect, thanks for working on this!
